### PR TITLE
Cc backend setup

### PR DESCRIPTION
**Pull request by OpaJaa**
Skeleton server for the backend. Starts with "npm start" and uses port 3000.

### CODE REVIEW

**Comment by rvsape** [comment]
add .gitignore and ignore node_modules/

**Comment by OpaJaa** [comment]
> add .gitignore and ignore node_modules/

Done. Used the one mentioned by Ali in his pull request. Didn't notice we lacked one and just committed and pushed all... Sorry! :D

**Comment by bunjovis** [comment]
As @rvsape commented in Teams chat, Typescript support should also be added already

**Comment by OpaJaa** [comment]
> As @rvsape commented in Teams chat, Typescript support should also be added already

ts support added using the tutorial @rvsape linked to chat.

**Comment by bunjovis** [comment]
I think this is still missing an example dotenv file (.env.example for example), which contains all the necessary env configuration parameters with no secrets. Also something in readme about creating the .env file before it can be run.

**Comment by OpaJaa** [comment]
> I think this is still missing an example dotenv file (.env.example for example), which contains all the necessary env configuration parameters with no secrets. Also something in readme about creating the .env file before it can be run.

Added .env.example. It only has the port for now. Also added better instructions to readme. Let me know, if the instructions have something wrong in them or need something more added.

File: cc-backend/readme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- Clone from repo.
```
Comment (rvsape) [inline cc-backend/readme.md]: some mention of the technologies used, like "Express Typescript server"
formatting guide: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by rvsape** [comment]
need to come up with some consensus about the ts - configurations in every service but its a problem for another PR

File: cc-backend/readme.md (hunk 1 above)
Comment (OpaJaa) [reply cc-backend/readme.md]: Updated readme with some technologies and versions. Should discuss together what and add there and how we format them so every readme would look similar.

**Review by rvsape — APPROVED** [review]

**Review by bunjovis — APPROVED** [review]

**Pull request merged**